README.rst

```diff
@@ -171,10 +179,15 @@ if you are interested in a higher success rate.
 
     python -m autoextract urls.txt --page-type articles --max-query-error-retries 3 --output res.jl
 
-Queries with success will be buffered while failing queries will be retried
-until the max number of retries or a timeout is reached. If it's not possible
-to successfully retrieve all queries, we'll return the last available results
-including both successes and failures.
+Failing queries are retried
+until the max number of retries or a timeout is reached.
+If it's still not possible to fetch all queries without errors,
+the last available result is written to the output
+including both queries with success and the ones with errors.
+
+When batches are used,
```

Comment:
IMO this is an implementation detail. It is a comment I would include the in docstring of the methods, but not in the README. As is, it would be very confusing for the reader. A CMD user wouldn't understand it, and also It wouldn't extract any practical conclusion about how this is affecting him. I propose just to remove the paragraph.

Reply:
What do you think, @kmike?

Reply:
I'm fine with @ivanprado's suggestion. I was on fence between suggesting to remove this section and suggesting to put more effort to make it more precise. Both sound good to me.

Reply:
Removing then 🚀